pallets/providers/src/lib.rs

```diff
@@ -276,54 +280,70 @@ pub mod pallet {
 
         /// Event emitted when a SP has changed is total data (stake) successfully. Provides information about
         /// that SP's account id, its old total data that could store, and the new total data.
```

Comment:
should this comment be updated to reflect the new nomenclature (total data/stake -> capacity)?

Reply:
Yep! Missed that one, ty!